kolibri/core/logger/test/test_api.py

```diff
@@ -133,101 +133,101 @@ def tearDown(self):
+#         response = self.client.post(reverse('kolibri:core:contentsummarylog-list'), data=self.payload, format='json')
+#         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+#     def test_anonymous_user_cannot_create_summarylog_for_learner(self):
+#         response = self.client.post(reverse('kolibri:core:contentsummarylog-list'), data=self.payload, format='json')
+#         self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+#     def test_anonymous_user_cannot_create_summarylog(self):
+#         del self.payload['user']
+#         response = self.client.post(reverse('kolibri:core:contentsummarylog-list'), data=self.payload, format='json')
+#         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+#     def test_user_log_filtering(self):
+#         self.client.login(username=self.admin.username, password=DUMMY_PASSWORD, facility=self.facility)
+#         response = self.client.get(reverse('kolibri:core:contentsummarylog-list'), data={"user_id": self.user2.id})
+#         expected_count = ContentSummaryLog.objects.filter(user__pk=self.user2.id).count()
+#         self.assertEqual(len(response.data), expected_count)
+
+#     def test_facility_log_filtering(self):
+#         response = self.client.login(username=self.superuser.username, password=DUMMY_PASSWORD)
+#         # add user3 to new facility
+#         self.facility2 = FacilityFactory.create()
+#         self.user3 = FacilityUserFactory.create(facility=self.facility2)
+#         [ContentSummaryLogFactory.create(user=self.user3, content_id=uuid.uuid4().hex,
+#                                          channel_id=uuid.uuid4().hex) for _ in range(1)]
+#         response = self.client.get(reverse('kolibri:core:contentsummarylog-list'), data={"facility": self.facility2.id})
+#         expected_count = ContentSummaryLog.objects.filter(user__facility_id=self.facility2.id).count()
+#         self.assertEqual(len(response.data), expected_count)
+
+#     def test_classroom_log_filtering(self):
+#         self.client.login(username=self.admin.username, password=DUMMY_PASSWORD, facility=self.facility)
+#         response = self.client.get(reverse('kolibri:core:contentsummarylog-list'), data={"classroom": self.classroom.id})
+#         expected_count = ContentSummaryLog.objects.filter(user__pk=self.user2.id).count()
+#         self.assertEqual(len(response.data), expected_count)
+
+#     def test_learner_group_log_filtering(self):
+#         self.client.login(username=self.admin.username, password=DUMMY_PASSWORD, facility=self.facility)
+#         response = self.client.get(reverse('kolibri:core:contentsummarylog-list'), data={"learner_group": self.learner_group.id})
+#         expected_count = ContentSummaryLog.objects.filter(user__pk=self.user2.id).count()
+#         self.assertEqual(len(response.data), expected_count)
+
+#     def tearDown(self):
+#         self.client.logout()
```

Comment:
@jredrejo we usually recommend deleting code rather than commenting it out, unless you expect this to be very temporary

Reply:
Yep, it's a temp change to try to reproduce this problem in Travis. I will restore it tomorrow

Reply:
coolio